services/QuillLMS/app/views/application/_sub_header.html.erb

```diff
@@ -14,7 +14,7 @@
     </div>
   <% else %>
     <div class="logo-and-badge-container">
-      <a aria-label="Quill" class="focus-on-dark" href="<%= root_path %>">
+      <a aria-label="Quill" class="focus-on-light" href="<%= root_path %>">
```

Comment:
I think it may be the case that we had `focus-on-dark` here because it renders on the logged-out homepage as well -- we may need to make this a conditional the same we do for the `logo_src` ?